### ISSUE TITLE
Document how to use CMake with only regular find_XXX

### ISSUE THREAD

**Issue by lasote** [conan-io/docs#629]

**Comment by jgsogo** [comment]
I think that this issue could be related to the second part of this one (https://github.com/conan-io/conan/issues/4206). I would want to know how far we are from the @memsharded comment:

> Yes, exactly that is the idea of the future improvements to build-helpers and generators, to decouple the information from dependencies (include paths, library names) from information of build configuration (toolchain), so both could be used in a more idiomatic way by the build systems, leaving the build-helpers simpler and doing just the work of calling the build systems, but not providing any configuration on the command line.

I'm assigning this to myself because I want to investigate it a little bit more.

**Comment by jgsogo** [comment]
I'm not sure what should be said but https://docs.conan.io/en/latest/integrations/cmake/find_packages.html#find-packages

**Comment by danimtb** [comment]
Yes, I think that section covers this issue. However, maybe that section is not as visible as it may be regarding issues like this one conan-io/conan#4844

**Comment by jgsogo** [comment]
I would love to see the use case of that issue documented, or at least a working example on our repo, or maybe a blogpost... yes, I didn't understand that kind of CMake usage. 💯

**Comment by arnaudgelas** [comment]
@jgsogo If you want a real and larger example of this kind of CMake usage, you can have a look at https://github.com/kde-frameworks-conan/extra-cmake-modules

**Comment by danimtb** [comment]
That example is coming also to the conan-io/examples repo even for multiconfig debug/release

**Comment by danimtb** [comment]
We should include this in that section: https://github.com/conan-io/examples/tree/master/features/cmake/find_package/exported_targets_multiconfig
